### PR TITLE
Feat: Support text search in question options, Support multiple choice

### PR DESCRIPTION
**Pull request by winged**
Add support for searching multiple choice questions (both dynamic and
static). And while we're at it, also change the lookup of choice value
to use the option's label instead of the slug, which ought to be much
more useful.
    
Note: This implies that the label is present in the user's current language, otherwise it won't find possible matches.

### CODE REVIEW

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
NIce work! :tada:

Just two nitpicks.

Technically this should be considered a breaking change, right?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3.4"
 services:
   db:
-    image: postgres:9.6-alpine@sha256:6f0f33cfc90a54ca57f81c14c6c35fde77701c2becb4344f41c7dea1235d5f71
+    image: postgres:10.15-alpine
```
[the remark this answers is not in the document]
Comment (winged) [reply docker-compose.yml]: This was part of a conflict resolution and shouldn't be here in the first place

File: caluma/caluma_form/filters.py [hunk 2]
```diff
@@ -331,19 +334,58 @@ def _apply_filter(self, qs, value):
...
         return qs
 
+    def _word_lookup_for_question(self, question, word, lookup):
+        lang = translation.get_language()
+        if question.type in (
+            Question.TYPE_CHOICE,
+            Question.TYPE_DYNAMIC_CHOICE,
+            Question.TYPE_MULTIPLE_CHOICE,
+            Question.TYPE_DYNAMIC_MULTIPLE_CHOICE,
+        ):
```
[the remark this answers is not in the document]
Comment (winged) [reply caluma/caluma_form/filters.py]: You're right, fixed

**Comment by winged** [comment]
> Technically this should be considered a breaking change, right?

I thought about this as well. Technically, searching choice fields never worked correctly, anyway: The option slug often only "accidentally" matches the displayed value, especially in multilingual environments. So we could even think of that part as a "fix" instead. And Multiple Choice wasn't supported before either :-)

**Comment by open-dynaMIX** [comment]
> So we could even think of that part as a "fix" instead

Fair enough!

**Review by open-dynaMIX — APPROVED** [review]

**Comment by czosel** [comment]
Hmm, if I’m not mistaken this is very much a breaking change. We implement search in choice questions usually using dropdowns where the key is the option slug. This would probably break most of those. Cc @anehx

**Comment by winged** [comment]
> Hmm, if I’m not mistaken this is very much a breaking change. We implement search in choice questions usually using dropdowns where the key is the option slug. This would probably break most of those. Cc @anehx

But do you use the `searchAnswers` filter for this case, or the more suitable `hasAnswer` one`? For strict lookups like when using dropdowns, the latter should be preferred anyway, as the search filter defaults to infix, which in turn could also return slugs that are superstring (opposite of substring??) of the requested one (Think `foo-bar` and `foo-bar-baz`, which would both be returned if you select `foo-bar`)

**Comment by czosel** [comment]
Ah right, seems that we're using `hasAnswer` in those cases. Thanks!

**Pull request merged**